<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menuWindow</class>
 <widget class="QDialog" name="menuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QWidget" name="bgWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgWidget{
background-color: qlineargradient(spread:pad, x1:0.073, y1:0.128341, x2:1, y2:1, stop:0 rgba(209, 107, 165, 255), stop:1 rgba(255, 255, 255, 255));}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Monospace&quot;;</string>
    </property>
    <property name="text">
     <string>MENU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
border-color: rgb(255, 255, 255);
background-color: rgba(209, 107, 165, 255);
</string>
    </property>
    <property name="text">
     <string>CADASTRAR</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::InsertImage">
      <normaloff>Icon/signUp.png</normaloff>
      <normalon>Icon/signUp.png</normalon>
      <disabledoff>Icon/signUp.png</disabledoff>
      <disabledon>Icon/signUp.png</disabledon>
      <activeoff>Icon/signUp.png</activeoff>
      <activeon>Icon/signUp.png</activeon>
      <selectedoff>Icon/signUp.png</selectedoff>
      <selectedon>Icon/signUp.png</selectedon>Icon/signUp.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
border-color: rgb(255, 255, 255);
background-color: rgba(209, 107, 165, 255);</string>
    </property>
    <property name="text">
     <string>PESQUISAR</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentPrintPreview">
      <normaloff>Icon/search.png</normaloff>
      <normalon>Icon/search.png</normalon>
      <disabledoff>Icon/search.png</disabledoff>
      <disabledon>Icon/search.png</disabledon>
      <activeoff>Icon/search.png</activeoff>
      <activeon>Icon/search.png</activeon>
      <selectedoff>Icon/search.png</selectedoff>Icon/search.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
border-color: rgb(255, 255, 255);
background-color: rgba(209, 107, 165, 255);</string>
    </property>
    <property name="text">
     <string>SOBRE</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::FormatTextDirectionLtr">
      <normalon>Icon/about.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
border-color: rgb(255, 255, 255);
background-color: rgba(209, 107, 165, 255);</string>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ListRemove">
      <normalon>Icon/exit.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
